cvat-ui/src/components/header/header.tsx

```diff
@@ -33,7 +33,7 @@ import Select from 'antd/lib/select';
 import { getCore } from 'cvat-core-wrapper';
 import config from 'config';
 
-import { CVATLogo } from 'icons';
+import { CVATLogo, UpgradeIcon } from 'icons';
```

Comment:
I believe `Upgrade` work and icon looks unclear. Probably `Switch to Pro plan` with `$` icon should look better. Do you agree? Probably need to talk with Breeze.